.github/workflows/autoformat.yml

```diff
@@ -80,6 +80,11 @@ jobs:
           git fetch origin ${{ needs.check-comment.outputs.base_ref }}
           git merge origin/${{ needs.check-comment.outputs.base_ref }}
       # ************************************************************************
+      # Prettier
+      # ************************************************************************
+      - run: |
+          pre-commit run prettier --all-files --color=always
+      # ************************************************************************
```

Comment:
We can autoformat the files that need to be formatted using `@mlflow-automation autoformat`.

Reply:
This step doesn't take long. It's should be fine to always run this step.